app/models/dms.rb

```diff
@@ -29,13 +29,13 @@ def document_id
   def link
     return if @dms.blank?
     address = Geocodr.address_dms(@issue)
-    I18n.interpolate @dms[:create_link][@ddc],
-      ks_id: @issue.id,
-      ks_user: Current.user.login,
-      ks_str: "#{address['strasse_name']} (#{address['strasse_schluessel']} - #{address['gemeindeteil_name']})",
-      ks_hnr: address['hausnummer'],
-      ks_hnr_z: address['hausnummer_zusatz'],
-      ks_eigentuemer: @issue.property_owner.truncate(254, omission: '…')
+    @dms[:create_link][@ddc].gsub('{ks_id}', @issue.id.to_s).gsub('{ks_user}', Current.user.login)
+      .gsub('{ks_str}',
+        "#{address['strasse_name']} (#{address['strasse_schluessel']} - #{address['gemeindeteil_name']})")
+      .gsub('{ks_hnr}', address['hausnummer'])
+      .gsub('{ks_hnr_z}', address['hausnummer_zusatz'].presence || '')
+      .gsub('{ks_eigentuemer}',
+        @issue.property_owner.present? ? @issue.property_owner.truncate(254, omission: '…') : '')
```

Comment:
```suggestion
  I18n.interpolate @dms[:create_link][@ddc],
    ks_id: @issue.id,
    ks_user: Current.user.login,
    ks_str: "#{address['strasse_name']} (#{address['strasse_schluessel']} - #{address['gemeindeteil_name']})",
    ks_hnr: address['hausnummer'],
    ks_hnr_z: address['hausnummer_zusatz'],
    ks_eigentuemer: @issue.property_owner.truncate(254, omission: '…') 
```
Durch Anpassung der I18n Interpolation Pattern kann die Ersetzung auch für Werte wie `{key}` erfolgen. Siehe dazu auch den ergänzten I18n Initializer.